package.json

```diff
@@ -46,7 +46,8 @@
           "tab-scroller",
           "infrastructure",
           "tab-indicator",
-          "text-field"
```

Comment:
also add mdc-typography as a dep.

Reply:
I have added `typography` (on line _50_) already.

I also see that `"@material/typography": "^0.40.1"` was already on line _80_ of this file.

Anything I am missing?

Reply:
AH! I meant `"@material/typography": "^0.40.1"`...thanks